### PR TITLE
Use ember-metrics to handle Google Analytics

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
Takes care of async loading of ga.js & disables on ember 'dev' & 'test' environments.

### CODE REVIEW

File: app/mixins/google-pageview.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-        if (!window.ga) {
-            return;
-        }
-        return window.ga('send', 'pageview', {
-            page: this.get('url'),
-            title: this.get('url')
+        Ember.run.scheduleOnce('afterRender', this, () => {
+            const page = this.get('url');
+            const title = this.getWithDefault('currentRouteName', 'unknown');
+            Ember.get(this, 'metrics').trackPage({ page, title });
```
Comment (Turbo87) [inline app/mixins/google-pageview.js]: Is the afterRender scheduling necessary here?

File: app/mixins/google-pageview.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
     notifyGoogleAnalytics: Ember.on('didTransition', function() {
-        if (!window.ga) {
-            return;
-        }
-        return window.ga('send', 'pageview', {
-            page: this.get('url'),
-            title: this.get('url')
+        Ember.run.scheduleOnce('afterRender', this, () => {
+            const page = this.get('url');
+            const title = this.getWithDefault('currentRouteName', 'unknown');
```
Comment (Turbo87) [inline app/mixins/google-pageview.js]: `title` had been set to `this.get('url')` too until now. maybe we should keep it like that for now and change that in a separate PR?

File: app/mixins/google-pageview.js (hunk 1 above)
Comment (sivakumar-kailasam) [inline app/mixins/google-pageview.js]: Is it an overkill? Thought I'd keep it there for prioritising the route load over ga stuff

File: app/mixins/google-pageview.js (hunk 2 above)
Comment (sivakumar-kailasam) [inline app/mixins/google-pageview.js]: Yeah thought I'd get opinions on the PR on this. Will switch to current values

File: app/mixins/google-pageview.js (hunk 1 above)
Comment (Turbo87) [inline app/mixins/google-pageview.js]: just noticed that it's mentioned in the ember-metrics README like this, so it's probably fine. still wondering if it's really still needed though.

**Pull request merged**